src/permcheck.cu

```diff
@@ -62,7 +114,12 @@ void make_vec_map_c(int n,
       perm_map[row_s + j]  = row_s + j;
       perm_cols[row_s + j] = rev_perm[cols[row_s + j]];
     }
+#if 0
     selection_sort2(rowlen, &perm_cols[row_s], &perm_map[row_s]);
+#else
+    //quickSort(&perm_cols[row_s], &perm_map[row_s], 0, rowlen-1);
+    insertion_sort(rowlen, &perm_cols[row_s], &perm_map[row_s]);
```

Comment:
I have a suggestion for an alternative sort. Nothing to change here, but I think I should create an issue for it once I'm back at my computer...

Reply:
This is probably outside the scope of this PR, so I will resolve this conversation, but I look forward to the issue on alternative sort options.